docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,289 @@
+          icon: "elasticStack",
+        },
+        {
+          name: "Panther",
+          href: "./export-audit-events/panther/",
+          icon: "panther",
+        },
+        {
+          name: "Datadog",
+          href: "./export-audit-events/datadog/",
+          icon: "datadog",
+        },
+        {
+          name: "Splunk",
+          href: "./export-audit-events/splunk/",
+        },
+      ]
+    },
+    {
+      title: "Self-Host Teleport",
+      description: "Run Teleport Enterprise in your own infrastructure, with guides covering high availability and multi-region clusters, securing key material with KMS or HSMs, and more.",
+      href: "./deploy-a-cluster/",
+      tags: [
+        {
+          name: "AWS KMS",
+          href: "./deploy-a-cluster/aws-kms/",
+          icon: "aws",
+        },
+        {
+          name: "Google Cloud KMS",
+          href: "./deploy-a-cluster/gcp-kms/",
+          icon: "googleCloud",
+        },
+        {
+          name: "Helm Chart",
+          href: "./deploy-a-cluster/helm-deployments/",
+        },
+      ]
+    },  
+    {
+      title: "Infrastructure as Code",
+      description: "Manage Teleport's Dynamic Resources using infrastructure as code tools, including Terraform, Helm and the Teleport tctl client tool.",
+      href: "./infrastructure-as-code/",
+      tags: [
+        {
+          name: "Terraform",
+          href: "./infrastructure-as-code/terraform-provider/",
+        },
+        {
+          name: "Kubernetes Operator",
+          href: "./infrastructure-as-code/teleport-operator/",
+        },
+        {
+          name: "Helm Chart",
+          href: "./infrastructure-as-code/teleport-operator/teleport-operator-helm/",
+        },
+      ]
+    },
+    {
+      title: "Security Best Practices",
+      description: "Run Teleport Enterprise in your own infrastructure, with guides covering high availability and multi-region clusters, secure CA keys with KMS or HSM, and more.",
+      href: "./management/security/",
+      tags: [
+        {
+          name: "Reducing the Blast Radius of Attacks",
+          href: "./management/security/reduce-blast-radius/",
+        },
+        {
+          name: "Restrict Access",
+          href: "./management/security/restrict-privileges/",
+        },
+        {
+          name: "Revoking Access",
+          href: "./management/security/revoking-access/",
+        },
+      ]
+    }
+  ]}
+/>
+
+<UseCasesList
+  title="Use Cases"
+  useCases={[
+    {
+      title: "Enroll and Protect Your Infrastructure",
+      description: "Apps, servers, databases, Kubernetes, desktops, & more",
+      href: "../enroll-resources/",
+    },  
+    {
+      title: "VNet: Build without VPNs",
+      description: "Secure app & SSH access with no VPNs or proxies",
+      href: "../enroll-resources/application-access/guides/vnet/",
+    },
+    {
+      title: "Secure MCP (Protect the Vibes)",
```

Comment:
I do love the "Protect the Vibes" but it feels more appropriate for a marketing page than for product docs.